src/traces/scatter/attributes.js

```diff
@@ -283,7 +290,8 @@ module.exports = {
             role: 'info',
             description: [
                 'Has only an effect if `marker.color` is set to a numerical array.',
-                'Sets the lower bound of the color domain.'
+                'Sets the lower bound of the color domain.',
```

Comment:
Since we're making this *super* explicit (and using `cmin` AND `cmax` was mentioned in the issue: https://github.com/plotly/plotly.js/issues/331#issuecomment-196514964) - I would vote to add: must be used w/ `cmax` here as well.